main.go

```diff
@@ -29,12 +29,13 @@ import (
 )
 
 var (
-	bqDataset  = flag.String("bq_dataset", "", "DataCommons BigQuery dataset.")
-	btTable    = flag.String("bt_table", "", "DataCommons Bigtable table.")
-	btProject  = flag.String("bt_project", "", "GCP project containing the BigTable instance.")
-	btInstance = flag.String("bt_instance", "", "BigTable instance.")
-	projectID  = flag.String("project_id", "", "The cloud project to run the mixer instance.")
-	port       = flag.String("port", ":12345", "Port on which to run the server.")
+	bqDataset    = flag.String("bq_dataset", "", "DataCommons BigQuery dataset.")
+	btTable      = flag.String("bt_table", "", "DataCommons Bigtable table.")
+	btProject    = flag.String("bt_project", "", "GCP project containing the BigTable instance.")
+	btInstance   = flag.String("bt_instance", "", "BigTable instance.")
+	projectID    = flag.String("project_id", "", "The cloud project to run the mixer instance.")
+	branchFolder = flag.String("branch_folder", "", "The branch cache gcs folder.")
```

Comment:
Is it just for testing?

Reply:
Yes, this is for local testing with custom branch folder